src/ts/cmd/endpoint-registry/src/utils/httpErrors.ts

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
Maybe a module is the better way to go. Not sure if we should write all this classes by our own. 
https://www.npmjs.com/package/http-errors

Reply:
Indeed we can consider using something like this. We started initially with our own to have more control on error handling and reporting. I will make a issue in jira